cron/disabled/reject-contributions.js

```diff
@@ -50,6 +50,7 @@ async function run({ dryRun = false } = {}) {
     const order = await models.Order.findByPk(row['id']);
     const collective = await models.Collective.findByPk(order.CollectiveId);
     const fromCollective = await models.Collective.findByPk(order.FromCollectiveId);
+    const fromCollectiveUser = await models.User.findByPk(fromCollective.CreatedByUserId);
```

Comment:
`CreatedByUserId` is meant for reporting/logging purposes and should be avoided for this kind of things. Plus with this code we'd be recording in the database that the user who created the collective is the one who triggered the refund, which is not true. A lot of collectives have Pia or Xavier as the `CreatedByUserId` because they helped collectives to get setup, we don't want their names to be associated with the refunds.

What would make sense for the data model would be to allow user to be `null` in `refundTransaction`, which would mean a transaction "created by no user in particular", or rather "created by Open Collective's system". We may need to convert some `CreatedByUserId` columns to nullable to achieve that.

Reply:
Ok, will change that today.

Reply:
This seems like it could be quite a large change for how `refundTransaction` works. @znarf would there be any issue with this?

Reply:
I agree with @Betree that it should be ok to have a refund without a `CreatedByUserId`